raiden/network/proxies/token_network.py

```diff
@@ -300,30 +302,24 @@ def new_netting_channel(
 
         return channel_identifier
 
-    def _inspect_channel_identifier(
+    def get_channel_identifier(
             self,
             participant1: Address,
             participant2: Address,
-            called_by_fn: str,
             block_identifier: BlockSpecification,
-            channel_identifier: ChannelID = None,
     ) -> ChannelID:
-        if not channel_identifier:
```

Comment:
Allowing `channel_identifier` effectively made `_inspect_channel_identifier` into two functions, one that returns the latest channel id and another that validates the input for proxies.

These two are completely unrelated functionalities, so I just removed the argument and do the value checking at the proper method, the problem here is that another level of indirection was added for a very silly feature, which makes following the code a bit harder. This is IMO an example why sharing code blindfolded is not really a good idea.